action.yml

```diff
@@ -36,19 +36,24 @@ inputs:
     description: 'If true, then the action will not expect the commits to have a verification signature. It is required to set this to true in GitHub Enterprise Server'
     required: false
     default: false
+  skip-verification:
+    type: boolean
+    description: 'If true, the action will not validate the user or the commit verification status'
+    default: false
 
 runs:
   using: 'composite'
   steps:
     - name: Fetch metadata
       id: dependabot-metadata
       uses: dependabot/fetch-metadata@v1
-      if: ${{ github.event_name == 'pull_request' && github.actor == 'dependabot[bot]' }}
+      if: github.event_name == 'pull_request' && (github.actor == 'dependabot[bot]' || skip-verification == 'true')
       with:
         skip-commit-verification: ${{ inputs.skip-commit-verification }}
+        skip-verification :  ${{ inputs.skip-verification }}
```

Comment:
See https://github.com/dependabot/fetch-metadata/blob/main/action.yml#L20-L23

Reply:
I am not sure i understand what you mean here. We're not making any decision here as all we're doing is to pass the parameter down to fetch-metadata

Reply:
Yes, it must be executed since we required the data return from `fetch-metadata`.

Reply:
I don't understand this folks, apologies. Why do we have the condition in the "if: "? don't we want to invoke the fetch-metadata action regardless of whether skip-verification is true or false?